rules/do-you-have-the-best-github-issue-template/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+archivedreason:
+title: Do you have the best GitHub Issue template
+guid: ba8f3441-139a-4638-9fc8-144408902104
+uri: do-you-have-the-best-github-issue-template
+created: 2021-07-06T12:44:34.0000000Z
+authors:
+  - title: Tom Bui
+    url: https://www.ssw.com.au/people/tom-bui
+related:
+- do-you-create-a-sprint-forecast-aka-the-functionality-that-will-be-developed-during-the-sprint
+
+---
+### Do you have the best GitHub Issue template
+
+GitHub issues (classic) offers a great way of raising issues within the projects. However, it can be difficult to distinguish whether the issue is about a bug fix, feature request or just a topic question.
+Therefore, GitHub issue templates should be used to help standardize and categorize different issue queries
+
+<!--endintro-->
+
+Based on [Configuring issue templates for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
+
+To create GitHub issues templates
+
+1. On GitHub, navigate to the main page of the repository.
+
+2. Under your repository name, click  Settings.
+![image](https://user-images.githubusercontent.com/67776356/124532033-0efc4e00-de53-11eb-9013-f43428e9dead.png)
```

Comment:
looks like you are referencing someone else's images - we should have our own to avoid breaking images

Reply:
Done - updated to use own images

Reply:
I can't see the files in the commit history - it looks like they are still using https://user-images.githubusercontent.com/

Reply:
Done - changed image folder sources